packages/spindle-ui/src/NavigationTab/UnderlineTab/UnderlineTab.css

```diff
@@ -0,0 +1,207 @@
+}
+
+.spui-UnderlineTab-button:focus {
+  border-radius: 4px;
+  outline: 2px solid var(--color-focus-clarity);
+  outline-offset: -2px;
+}
+
+.spui-UnderlineTab-button:not(:focus-visible) {
+  outline: none;
+}
+
+.spui-UnderlineTab-labelWrapper {
+  align-items: center;
+  column-gap: 6px;
+  display: flex;
+  margin: 0 auto;
+  max-width: 100%;
+  padding: 14px 0;
+  position: relative;
+  width: fit-content;
+}
+
+.spui-UnderlineTab-labelWrapper::after {
+  background-color: var(--color-border-accent-primary);
+  border-radius: 3px;
+  bottom: 0;
+  content: '';
+  height: 3px;
+  min-width: 40px;
+  opacity: 0;
+  position: absolute;
+  transition: opacity 0.35s ease-in-out;
+  width: 100%;
+}
+
+/* stylelint-disable-next-line  */
+.spui-UnderlineTab-button[aria-selected='true'] .spui-UnderlineTab-labelWrapper::after {
+  opacity: 1;
+}
+
+.spui-UnderlineTab-label {
+  color: var(--color-text-low-emphasis);
+  font-size: 0.875rem;
+  font-weight: bold;
+  line-height: 1.3;
+}
+
+.spui-UnderlineTab--fixed .spui-UnderlineTab-label {
+  min-width: 0;
+  overflow-wrap: break-word;
+}
+
+/* stylelint-disable-next-line  */
+.spui-UnderlineTab-button[aria-selected='true'] .spui-UnderlineTab-label {
+  color: var(--color-text-accent-primary);
+}
+
+.spui-UnderlineTab-badge {
+  background-color: var(--color-surface-accent-primary);
+  border-radius: 8px;
+  box-sizing: border-box;
+  color: var(--color-text-high-emphasis-inverse);
+  flex-shrink: 0;
+  font-size: 0.6875rem;
+  font-weight: bold;
+  line-height: 1.3;
+  min-width: 20px;
+  padding: 1px 4px;
+}
+
+.spui-UnderlineTab-visuallyHidden {
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+}
+
+.spui-UnderlineTab-arrow {
+  background-color: var(--color-surface-primary);
+  bottom: 0;
+  height: fit-content;
+  margin: auto;
+  opacity: 0;
+  position: absolute;
+  top: 0;
+  transition:
+    opacity 0.35s ease-in-out,
+    visibility 0.35s ease-in-out;
+  visibility: hidden;
+  z-index: 1;
+}
+
+.spui-UnderlineTab-arrow.is-showed {
+  opacity: 1;
+  visibility: visible;
+}
+
+.spui-UnderlineTab-arrow::before {
+  content: '';
+  height: 100%;
+  position: absolute;
+  width: 10px;
+  z-index: -1;
+}
+
+.spui-UnderlineTab-arrow--left {
+  left: 0;
+}
+
+.spui-UnderlineTab-arrow--left::before {
+  background: linear-gradient(
+    270deg,
+    rgba(255, 255, 255, 0) 0,
+    var(--color-surface-primary) 100%
+  );
+  left: 42px; /* ボタンの横幅分 */
+}
+
+.spui-UnderlineTab-arrow--right {
+  right: 0;
+}
+
+.spui-UnderlineTab-arrow--right::before {
+  background: linear-gradient(
+    270deg,
+    var(--color-surface-primary) 0,
+    rgba(255, 255, 255, 0) 100%
+  );
+  right: 42px; /* ボタンの横幅分 */
+}
+
+.spui-UnderlineTab-arrowButton {
+  background-color: transparent;
+  border: none;
+  border-radius: 8px;
+  color: var(--color-object-low-emphasis);
+  display: flex;
+  padding: 13px;
```

Comment:
別件のコメント中に気づいてしまったのですが、<>ボタンのタップ領域が42pxしかないので、44pxにするために
```suggestion
  padding: 14px;
```
もしくは15pxにすると良いかもしれません。
<img width="328" alt="image" src="https://github.com/openameba/spindle/assets/4669600/617035e0-7fb7-45c9-a7d1-881dc589164c">

とはいえ今回のアクセシビリティ要件に[ターゲットサイズ](https://a11y-guidelines.ameba.design/2/5/5/)は含まれていないので、必須ではないです😀

Reply:
@tatz-ibz  そういえば、ターゲットサイズをアクセシビリティ要件に含めなかったのは、リストのitemの幅が44px担保できないかもだからなんですっけ・・？？👀

imo. figmaを見る限り、item幅48px・高さ46px、<>ボタンが幅48px・高さ46pxのアウトラインサイズなので、44px担保できない・・とデザイン上考えているようには見えませんでした。（二文字であれば44px未満にはなりますが、普通に押しづらいわけですし。文字数１文字はユースケースとして考えづらいので除外）
リストのitemの最低幅を44pxにすることがデザイン的に問題なければ、ターゲットサイズもアクセシビリティ要件に含めることができるのではと思い、どうでしょうか？？（DesignDocの前提ひっくり返しで申し訳ない:pray:）
<img width="676" alt="image" src="https://github.com/openameba/spindle/assets/4669600/12fc0523-98f1-42bb-a55a-4970458ddb35">
<img width="427" alt="image" src="https://github.com/openameba/spindle/assets/4669600/761695db-0a61-4b7d-b5ca-31bd7c1c40a4">


2文字だと36px幅になる
<img width="382" alt="image" src="https://github.com/openameba/spindle/assets/4669600/50cc3e51-0680-4779-8d8d-9084a09c07cd">

Reply:
44px担保するように作ってるっぽいです!! by @MasatoHonda

Reply:
かっけぇ・・

Reply:
（念の為、のちさんの返答を待ちます :loading:）

Reply:
（のちさんお休みでした 🙏🏻 ）

> @tatz-ibz そういえば、ターゲットサイズをアクセシビリティ要件に含めなかったのは、リストのitemの幅が44px担保できないかもだからなんですっけ・・？？👀

おそらくですが、Figma見ると左右移動ボタンが42*42で組まれていたので要件から削ったのかな？と思います 🤔 （多分）

![スクリーンショット 2024-02-05 15 12 07](https://github.com/openameba/spindle/assets/42470015/2db190d3-10ef-47d6-a628-0962228bd3bb)

ただ、デザイナーさん的にも意図してないとのことなので44*44になるように修正し、アクセシビリティ要件にも追加しました！ https://github.com/openameba/spindle/pull/909/commits/de1c9e1691c09e220f365bbabc95e724fc8ebbf7

Reply:
ほんとだ！なるほどw
調査＆修正ありがとうございます😁

要件にも追加いただいたので、
```
diff --git a/packages/spindle-ui/src/NavigationTab/UnderlineTab/UnderlineTab.css b/packages/spindle-ui/src/NavigationTab>
index 0ea7d45..e6b5cf5 100644
--- a/packages/spindle-ui/src/NavigationTab/UnderlineTab/UnderlineTab.css
+++ b/packages/spindle-ui/src/NavigationTab/UnderlineTab/UnderlineTab.css
@@ -60,21 +60,28 @@
   width: fit-content;
 }

+.spui-UnderlineTab--scrollable .spui-UnderlineTab-button {
+  min-width: 44px;
+}
+
 .spui-UnderlineTab-labelWrapper::after {
   background-color: var(--color-border-accent-primary);
   border-radius: 3px;
   bottom: 0;
   content: '';
   height: 3px;
-  min-width: 40px;
+  left: -3px;
   opacity: 0;
+  padding: 0 3px;
   position: absolute;
+  right: -3px;
   transition: opacity 0.35s ease-in-out;
   width: 100%;
 }
```
のようにタブitemのbutton幅を44px担保する方がよさそうです。たとえ1文字のタブitemでも44px以上にできます。
<img width="1406" alt="image" src="https://github.com/openameba/spindle/assets/4669600/3421044f-9769-4540-91c0-2ae82663509c">

Reply:
完璧です！！🎉